app/index.html

```diff
@@ -33,13 +34,13 @@
   <meta name="theme-color" content="#497495">
   <meta name="google" content="notranslate">
 
+  <meta property="og:image" content="/img/og-image.jpg">
```

Comment:
Must be "https://web.telegram.org/img/logo_share.png"

Reply:
done